fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java

```diff
@@ -1371,6 +1371,18 @@ private void sendResult(boolean isOutfileQuery, boolean isSendFields, Queriable
         }
         profile.getSummaryProfile().setQueryScheduleFinishTime();
         updateProfile(false);
+        if (coord.getInstanceTotalNum() > 1) {
+            try {
+                LOG.debug("Start to execute fragment. user: {}, db: {}, sql: {}, fragment instance num: {}",
```

Comment:
add  if  Log.isdebugenabled() because the sql maybe very long 
in some cases it may cost a lot of memory or cpu time.